### PR TITLE
Parent application

### PR DESCRIPTION
**Pull request by LenkVskv**

### CODE REVIEW

File: src/app/shell/personal-cabinet/applications/applications.component.html [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
-        *ngTemplateOutlet="Applications; context:{$implicit: applications | applicationFilter : 'approved' }">
-      </ng-container>
-    </mat-tab>
-    <mat-tab label="Відхилені">
-      <ng-container *ngTemplateOutlet="Applications; context:{$implicit: applications | applicationFilter : 'denied' }">
-      </ng-container>
-    </mat-tab>
-  </mat-tab-group>
+  <div *ngIf="role==='parent' then isParent; else isProvider"></div>
+    <ng-template #isParent>
```
Comment (dmitryy90) [inline src/app/shell/personal-cabinet/applications/applications.component.html:5]: indentations

File: src/app/shell/personal-cabinet/applications/applications.component.ts [hunk 2]
```diff
@@ -21,6 +23,12 @@ export class ApplicationsComponent implements OnInit {
   applications$: Observable<Application[]>;
   public applications: Application[];
   child: Child;
+  
+  @Select(RegistrationState.user)
+  user$: Observable<User>;
+  role;                                                                                                                                                 ;
```
Comment (dmitryy90) [inline src/app/shell/personal-cabinet/applications/applications.component.ts:29]: type?

File: src/app/shell/personal-cabinet/applications/applications.component.ts [hunk 3]
```diff
@@ -35,6 +43,9 @@ export class ApplicationsComponent implements OnInit {
       this.applications = applications
     );
 
+    this.user$.subscribe(user => this.role = user.role);
```
Comment (dmitryy90) [inline src/app/shell/personal-cabinet/applications/applications.component.ts:46]: you can use selectSnapshot to avoid additional subscription

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/shell/personal-cabinet/applications/applications.component.ts (hunk 3 above)
Comment (litvinets) [reply src/app/shell/personal-cabinet/applications/applications.component.ts:46]: @LenkVskv  see examples in these components 
image

**Review by dmitryy90 — APPROVED** [review]

**Pull request closed without merging**